doc/api_assets/style.css

```diff
@@ -330,6 +330,27 @@ hr {
   margin-top: .666em;
 }
 
+#toc .stability_0::after,
+#toc .stability_0 a:hover,
+#toc .stability_0 a:focus {
+  background-color: #d50027;
+  color: #fff;
+}
+
+#toc .stability_0::after {
```

Comment:
This may be better as `::before`. Domains and Punycode formats poorly with the current:

(There may be some other way if resoling it that I am forgetting though.)

<img width="685" alt="screen shot 2016-09-06 at 11 05 56 am" src="https://cloud.githubusercontent.com/assets/1093990/18279014/1a3d1c54-7422-11e6-9547-5d38fbdfd0c8.png">

<img width="548" alt="screen shot 2016-09-06 at 11 06 03 am" src="https://cloud.githubusercontent.com/assets/1093990/18279018/1fa0bc5a-7422-11e6-9d6c-10de78e07e18.png">